tox.ini

```diff
@@ -28,20 +30,19 @@ deps =
     ; Needed for `BBHx` package to work with PyCBC
     git+https://github.com/titodalcanton/BBHx.git@py39-and-cleanup; sys_platform == 'linux'
     git+https://github.com/gwastro/BBHX-waveform-model.git; sys_platform == 'linux'
-conda_deps=
-    mysqlclient
-    lin: gcc_linux-64>=12.2.0
-    lin: gxx_linux-64>=12.2.0
-    ; mac doesn't work, need fix
-    ; mac: clang_osx-64
-    ; mac: clangxx_osx-64
+conda_deps =
+    c-compiler
+    cxx-compiler
     gsl
-    lapack==3.6.1
-conda_channels=conda-forge
-setenv =
-    ; Tell the linker to look for shared libs inside the temporary Conda env.
-    ; Needed to build BBHx's wheel, whick links to LAPACK.
-    LIBRARY_PATH={envdir}/lib:{env:LIBRARY_PATH:}
+    ligo-segments
```

Comment:
@duncanmmacleod Thanks for looking into this!

A couple of these are python packages that we should be picking up through the requirements. At least

ligo-segments and python-ligo-lw should probably be removed from conda_deps, no?

Reply:
@ahnitz, these two packages are specifically called out in the conda requirements because PyPI only has a source distribution and these projects don't build cleanly with newer Python versions _and_ newer versions of Clang. This means that `pip install python-ligo-lw` fails.

The conda packages include the necessary patches to build, but that hasn't (yet) been backported into the source distribution.

Reply:
@duncanmmacleod Ok, that's fair enough. In which case, I'd just request a comment explaining this. In principle these lines should be removed once those changes have been included in the upstream code.

Reply:
@ahnitz, I have separated things out so that there is a unique commit that adds these packages to the `tox.ini` file (with a comment) that can be simply reverted when appropriate.